src/core/ext/filters/client_channel/lb_policy/round_robin/round_robin.c

```diff
@@ -535,9 +534,10 @@ static void rr_connectivity_changed_locked(grpc_exec_ctx *exec_ctx, void *arg,
         "[RR %p] connectivity changed for subchannel %p, subchannel_list %p: "
         "prev_state=%d new_state=%d p->shutdown=%d "
         "sd->subchannel_list->shutting_down=%d error=%s",
-        (void *)p, (void *)sd->subchannel_list, (void *)sd->subchannel,
-        sd->prev_connectivity_state, sd->curr_connectivity_state, p->shutdown,
-        sd->subchannel_list->shutting_down, grpc_error_string(error));
+        (void *)p, (void *)sd->subchannel, (void *)sd->subchannel_list,
+        sd->prev_connectivity_state, sd->pending_connectivity_state_unsafe,
```

Comment:
You could also use `grpc_connectivity_state_name()`, to make the log easier to read.